src/mdbExtensionController.ts

```diff
@@ -815,6 +816,41 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  async showSurveyForEstablishedUsers(): Promise<void> {
+    const surveyId = '9viN9wcbsC3zvHyg7';
+
+    const hasBeenShownSurveyAlready =
+      this._storageController.get(StorageVariables.GLOBAL_SURVEY_SHOWN) ===
+      surveyId;
+
+    // Show the overview page when it hasn't been show to the
+    // user yet, and they have saved connections
+    // -> they haven't just started using this extension
+    if (!hasBeenShownSurveyAlready) {
+      if (this._connectionStorage.hasSavedConnections()) {
```

Comment:
Total nit, we typically follow the [return early pattern](https://medium.com/swlh/return-early-pattern-3d18a41bba8) in our code.
In this case it would be:
```suggestion
    // Show the overview page when it hasn't been show to the
    // user yet, and they have saved connections
    // -> they haven't just started using this extension
    if (hasBeenShownSurveyAlready || !this._connectionStorage.hasSavedConnections()) {
      return;
    }
```

Reply:
ah yes, of course! thanks!